metricbeat/module/system/raid/mdinfo/mock.go

```diff
@@ -0,0 +1,50 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Ideally this should work if this file is moved to `mock_test.go`.

Reply:
Good idea!